doc/news/changes/minor/20180822BrunoTurcksin

```diff
@@ -0,0 +1,4 @@
+Fixed: mmult using TrilinosWrappers::SparseMatrix sometimes gave the wrong
+result in parallel 
```

Comment:
Write this as "TrilinosWrappers::SparseMatrix::mmult() sometimes gave the wrong result in parallel." to allow doxygen to link to the correct function.